newsroom/wire/views.py

```diff
@@ -142,7 +142,7 @@ def delete_dashboard_caches():
 def get_personal_dashboards_data(user, company, topics):
     def get_topic_items(topic):
         query = superdesk.get_resource_service("wire_search").get_topic_query(topic, user, company)
-        return list(superdesk.get_resource_service("wire_search").get_items_by_query(query, size=4))
+        return list(superdesk.get_resource_service("wire_search").get_items_by_query(query, size=6))
```

Comment:
Instead of hardcoding the size to 6, could we use the config for the card type itself, falling back to 6 if not defined?

Reply:
I've moved the config now to the backend so it knows which type is used and selects the size according to that